controllers/provider-gcp/pkg/internal/infrastructure/terraform.go

```diff
@@ -144,8 +144,31 @@ func RenderTerraformerChart(
 	config *api.InfrastructureConfig,
 	cluster *extensionscontroller.Cluster,
 ) (*TerraformFiles, error) {
+	var (
+		aggregationInterval string
+		flowSampling        float32
+		metadata            string
+	)
+
 	values := ComputeTerraformerChartValues(infra, account, config, cluster)
 
+	if config.Networks.FlowLogs != nil {
```

Comment:
Can you also implement a unit test for this case, please?

Reply:
This comment is not yet resolved, please check.

Reply:
Didn't we cover that with our unit tests when the flow log config was provided with wrong or correct values?

Reply:
This is a different code path. What should be unit tested here is that the Helm chart values are correctly computed before they are handed over to Terraform.